### PR TITLE
fix cairo-dock libs

### PR DESCRIPTION
**Pull request by Yisus7u7**

### CODE REVIEW

File: packages/cairo-dock-core/build.sh [hunk 1]
```diff
@@ -19,3 +20,19 @@ fi
...
+	elif [ $TERMUX_ARCH == "i686" ]; then
+		chmod +x ${TERMUX_PREFIX}/lib32/libgldi.so
+                chmod +x ${TERMUX_PREFIX}/lib32/cairo-dock/*
+		echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:$TERMUX_PREFIX/lib32" > $TERMUX_PREFIX/etc/profile.d/cairo-dock.sh
+	else
+		chmod +x ${TERMUX_PREFIX}/lib64/libgldi.so
+                chmod +x ${TERMUX_PREFIX}/lib64/cairo-dock/*
+		echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:$TERMUX_PREFIX/lib64" > $TERMUX_PREFIX/etc/profile.d/cairo-dock.sh
+	fi
+
```
Comment (xeffyr) [inline packages/cairo-dock-core/build.sh:37]: Please get rid of multilib sets. Termux doesn't support this. We also do not recommend using `LD_LIBRARY_PATH` anymore - it breaks stuff that uses Android libraries such as OpenSLES. All libraries **must** be installed into `$TERMUX_PREFIX/lib`. And don't set executable bit on libraries, especially from postinst script.

Termux file system layour: https://github.com/termux/termux-packages/wiki/Termux-file-system-layout

**Review by xeffyr — CHANGES_REQUESTED** [review]

**Comment by Yisus7u7** [comment]
@xeffyr Done, works fine.

**Comment by xeffyr** [comment]
> Done, works fine.

Not done. You need to get rid of `lib32` / `lib64`.

**Comment by Yisus7u7** [comment]
> > Done, works fine.
> 
> Not done. You need to get rid of `lib32` / `lib64`. Package source or build config should be modified to install binaries to `$TERMUX_PREFIX/lib`.

cairo-dock compulsorily uses these routes, it is even in the documentation of its official website

**Comment by Yisus7u7** [comment]
if you analyze the packages of other distro they respect this

**Comment by Yisus7u7** [comment]
@xeffyr 

https://glx-dock.org/ww_page.php?p=By%20compiling&lang=en

**Comment by Grimler91** [comment]
> if you analyze the packages of other distro they respect this

Well, we (as in termux) don't, installing 32bit packages on 64bit arches isn't supported, so there's no reason to split it into lib{32,64}

**Comment by Yisus7u7** [comment]
> > if you analyze the packages of other distro they respect this
> 
> Well, we (as in termux) don't, installing 32bit packages on 64bit arches isn't supported, so there's no reason to split it into lib{32,64}

**Comment by Grimler91** [comment]
> He did not understand me.
> 
> lib32 libraries will go on 32-bit devices, while 64-bit libraries will go on 64-bit devices, there are no lib32 libraries on 64-bit devices and vice versa

Yes, I can see that from the code. Are you suggesting that the libraries in all other packages should be moved to lib{32,64} as well? 

If not, then cairo-dock would be the only one out of 1000 packages that uses lib{32,64}. What makes cairo-dock special in this regard? 

If yes, then we would have to rebuild all packages and update all RPATHs. What would we gain from all this extra work?

**Comment by xeffyr** [comment]
> He did not understand me.

That's you do not understand me.

Termux doesn't create library directories `lib32` or `lib64`, i.e. directories specific to 32/64 bits. It uses `lib` instead, which will have 64 bit libraries in 64 bit devices and 32 bit on 32 bit devices. You shouldn't take care about placing library somewhere manually.

However if package makes `lib32` and `lib64`, that behavior must be patched to install library files directly to `$TERMUX_PREFIX/lib`.

Also: my availability is limited and therefore *I kindly request to submit changes that are complete, have reasonable quality and match Termux design* (https://wiki.termux.com/wiki/Differences_from_Linux). If package needs to be patched,then make patch. Pull request that do not implement requested changes will be closed.

> If not, then cairo-dock would be the only one out of 1000 packages that uses lib{32,64}. What makes cairo-dock special in this regard?

I remember there is (or was) a package that did same and was successfully patched to avoid installation to `lib64` or `lib32`. These directories are for multilib compatibility only. Termux doesn't support multilib for now.

**Comment by xeffyr** [comment]
Fixable by `-DLIB_SUFFIX=` as said in INSTALL file.

**Comment by xeffyr** [comment]
https://github.com/termux/x11-packages/commit/3500307822d756d8f1e1854dbfe77c123d5f068d

**Comment by Yisus7u7** [comment]
Oh ok, thanks!

**Pull request closed without merging**